packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -74,6 +74,11 @@ export interface TldrawEditorBaseProps {
 	 */
 	shapeUtils?: readonly TLAnyShapeUtilConstructor[]
 
+	/**
+	 * An array of migrations to apply to the editor's schema.
+	 */
+	migrations?: readonly MigrationSequence[]
```

Comment:
`TldrawEditorProps` has some separate props for whether we have a store or not. Could this go in the type-enforced "no store allowed" section?